test/IDE/complete_annotation.swift

```diff
@@ -0,0 +1,109 @@
+// GLOBAL_EXPR-DAG: Literal[_Image]/None:               <name>#imageLiteral</name>(<callarg><callarg.label>resourceName</callarg.label>: <callarg.type><typeid.sys>String</typeid.sys></callarg.type></callarg>); name=#imageLiteral(resourceName: String)
+// GLOBAL_EXPR-DAG: Literal[Tuple]/None:                (<callarg><callarg.param>values</callarg.param></callarg>); name=(values)
+// GLOBAL_EXPR-DAG: Keyword[#function]/None:            <name>#function</name>; name=#function
+// GLOBAL_EXPR-DAG: Decl[Module]/None:                  <name>Swift</name>; name=Swift
+// GLOBAL_EXPR-DAG: Decl[Struct]/OtherModule[Swift]:    <name>Int</name>; name=Int
+// GLOBAL_EXPR-DAG: Decl[FreeFunction]/OtherModule[Swift]: <name>print</name>(<callarg><callarg.label>_</callarg.label> <callarg.param>items</callarg.param>: <callarg.type>Any</callarg.type>...</callarg>, <callarg><callarg.label>to</callarg.label> <callarg.param>output</callarg.param>: &amp;<callarg.type><typeid.sys>TextOutputStream</typeid.sys></callarg.type></callarg>); name=print(items: Any..., to: &TextOutputStream)
+// GLOBAL_EXPR: End completions
+
+
+func testType(value: #^GLOBAL_TYPE^#) {}
+// GLOBAL_TYPE: Begin completions
+// GLOBAL_TYPE-DAG: Keyword/None:                       <keyword>Any</keyword>; name=Any
+// GLOBAL_TYPE-DAG: Decl[Struct]/CurrModule:            <name>MyStruct</name>; name=MyStruct
+// GLOBAL_TYPE-DAG: Decl[Module]/None:                  <name>swift_ide_test</name>; name=swift_ide_test
+// GLOBAL_TYPE-DAG: Decl[Module]/None:                  <name>Swift</name>; name=Swift
+// GLOBAL_TYPE-DAG: Decl[Struct]/OtherModule[Swift]:    <name>Int</name>; name=Int
+// GLOBAL_TYPE: End completions
+
+
+func testMember(value: MyStruct) {
+  value.#^EXPR_MEMBER^#
+}
+// EXPR_MEMBER: Begin completions, 7 items
+// EXPR_MEMBER-DAG: Keyword[self]/CurrNominal:          <keyword>self</keyword>; name=self
+// EXPR_MEMBER-DAG: Decl[InstanceVar]/CurrNominal:      <name>propNormal</name>; name=propNormal
+// EXPR_MEMBER-DAG: Decl[InstanceVar]/CurrNominal:      <name>propFunction</name>; name=propFunction
+// EXPR_MEMBER-DAG: Decl[InstanceMethod]/CurrNominal: <name>labelNameParamName</name>(<callarg><callarg.label>label</callarg.label> <callarg.param>param</callarg.param>: <callarg.type>(<keyword>inout</keyword> <typeid.sys>Int</typeid.sys>) <keyword>throws</keyword> -&gt; <typeid.user>MyStruct</typeid.user></callarg.type></callarg>)<keyword> rethrows</keyword>; name=labelNameParamName(label: (inout Int) throws -> MyStruct) rethrows
+// EXPR_MEMBER-DAG: Decl[InstanceMethod]/CurrNominal: <name>labelName</name>(<callarg><callarg.label>label</callarg.label>: <callarg.type>(<attribute>@autoclosure </attribute>() -&gt; <typeid.sys>Int</typeid.sys>) -&gt; <typeid.sys>Int</typeid.sys></callarg.type></callarg>); name=labelName(label: (@autoclosure () -> Int) -> Int)
```

Comment:
There is a space inside the attribute wrap, "<attribute\>@autoclosure </attribute\>", ideally the tag would not include the space.

Reply:
Same for `<keyword> rethrows</keyword>`. I will see if fixing this affects the performance. But to clarify, even if we don't fix this, it should not affect the clients practically, right?

Reply:
I can't say for sure either way.

Reply:
Done https://github.com/apple/swift/blob/339ebbfa89a3648220aeb765be0c19315d36f140/lib/IDE/CodeCompletionResultPrinter.cpp#L70-L84